ssv/spectest/generate/state_comparison/tests_MsgProcessingSpecTest/decide on slashable attestation.json

```diff
@@ -1033,7 +1033,7 @@
 								},
 								"StartValue": "AQIDBAUGBwgJCgECAwQFBgcICQoBAgMEBQYHCAkKAQIAAAAAAAAAAAECAwQFBgcICQoBAgMEBQYHCAkKAQIDBAUGBwgJCgECAQAAAAAAAAABAgMEBQYHCAkKAQIDBAUGBwgJCgECAwQFBgcICQoBAg=="
 						},
-						"DecidedValue": "AQIDBAUGBwgJCgECAwQFBgcICQoBAgMEBQYHCAkKAQIAAAAAAAAAAAECAwQFBgcICQoBAgMEBQYHCAkKAQIDBAUGBwgJCgECAQAAAAAAAAABAgMEBQYHCAkKAQIDBAUGBwgJCgECAwQFBgcICQoBAg==",
+						"DecidedValue": null,
```

Comment:
`DecidedValue` in BaseRunner has a misleading name.
If a value was decided on but in the post consensus stage it was rejected due to slashing checks, `BaseRunner.DecidedValue` will be null while `Instance.DecidedValue` won't be.

So I propose changing the name here to `Value` or something else...

Reply:
created #457